tests/data/instances/mc_tuples_with_background_instance_test.py

```diff
@@ -0,0 +1,284 @@
+# pylint: disable=no-self-use,invalid-name
+
+from unittest import TestCase
+import numpy as np
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.dataset import TextDataset
+from deep_qa.data.instances.mc_tuples_with_background_instance import MCTuplesWithBackgroundInstance, \
+    IndexedMCTuplesWithBackgroundInstance
+
+
+class TestMcTuplesWithBackgroundInstance(TestCase):
+
+    def setUp(self):
+        qIdx = 0
+        num_ans = 2
+        answerOptions = "cat|is|a mammal|context: which is a nocturnal mammal$$$cat|is|nocturnal|which " \
+                        "is a nocturnal mammal###snake|is|a mammal|which is a nocturnal " \
+                        "mammal$$$snake|is|nocturnal|which is a nocturnal mammal"
+        background = "cat|eat|food$$$cat|is|a mammal|with fur|which is a mammal$$$snake|is example " \
```

Comment:
it doesn't seem to like the + -- gives me a red underline.  I'm not sure when I can use one and when I can't.  Strong feelings?